charts/portal/README.md

```diff
@@ -503,23 +503,22 @@ The following table lists the configured parameters of the MySQL Subchart - http
 | `mysql.primary.configuration`   | MySQL Primary configuration to be injected as ConfigMap	   | `see values.yaml` |
 
 
-## Nginx-Ingress
-The following table lists the configured parameters of the Nginx-Ingress Subchart - https://github.com/helm/charts/tree/master/stable/nginx-ingress
```

Comment:
may be  https://github.com/kubernetes/ingress-nginx/tree/main/charts/ingress-nginx